test/vaadin-combo-box-dropdown.html

```diff
@@ -7,6 +7,9 @@
   <script src='../../webcomponentsjs/webcomponents-lite.js'></script>
   <link rel="import" href="../../test-fixture/test-fixture.html">
   <link rel="import" href="../src/vaadin-combo-box-dropdown.html">
+  <link rel="import" href="../src/vaadin-combo-box.html">
+  <link rel="import" href="../../vaadin-dialog/vaadin-dialog.html">
```

Comment:
Align with the combo-box import and use the unstyled version (/src) for dialog also

Reply:
changed